### PR TITLE
Docs seo

### LINKED ISSUE #309: Perform some search engine optimization for docs
Currently the documentation is hard to find on engines like google, duckgosearch, bing etc.
One has to enforce the inclusion of the term "weldx" by using quotation marks. Eventually we should somehow highlight this more.

### PR DESCRIPTION
**Pull request by marscher**
## Changes

added some meta data for the HTML documentation, add google site verification.

## Related Issues

Closes #309

## Checks

- [ ] updated CHANGELOG.md
- [ ] updated tests
- [ ] updated doc/
- [ ] update example/tutorial notebooks

### CODE REVIEW

File: .github/workflows/pytest.yml
Comment (marscher) [inline .github/workflows/pytest.yml:46]: Note that the default matplotlib backend used on Windows is somehow broken (TK library cannot be found for whatever reason). To work around this I forcefully use the Agg backend for this OS.

**Comment by marscher** [comment]
I'd love your feedback @CagtayFabry for the keywords, as they are most useful for the search engine optimization.

**Review by vhirtham — APPROVED** [review]
Don't have any noteworthy knowledge about SEO, so wait for Cagtays review before merging ;)

**Review by CagtayFabry — APPROVED** [review]

**Pull request merged**
